src/Table.tsx

```diff
@@ -82,26 +90,68 @@ export function Table<RowShape>({
           ))}
         </tr>
       </thead>
+
       <tbody>
-        {data.map((item, index) => (
-          <tr key={getRowKey(item)}>
-            {columns.map(
-              ({
-                render,
-                id
-              }) => (
-                <td 
-                  key={id}
-                  css={{
-                    // no border on the bottom row
-                    borderBottom: index === data.length-1 ? `none`: `1px solid ${colors.silver.dark}`,
-                    padding
-                  }}
-                >{render(item, index, data)}</td>
-              )
-            )}
-          </tr>
-        ))}
+        {loading
+          ? [{}, {}, {}, {}, {}].map((_, index) => (
+              <tr key={index}>
+                {columns.map(({ id }) => (
+                  <td
+                    key={id}
+                    css={{
+                      // no border on the bottom row
+                      borderBottom:
+                        index === data.length - 1
+                          ? `none`
+                          : `1px solid ${colors.silver.dark}`,
+                      paddingTop: '1rem',
+                      paddingBottom: '1rem',
+                      paddingRight: padding,
+                      paddingLeft: padding,
+                    }}
+                  >
+                    <div
+                      css={{
+                        backgroundColor: colors.silver.light,
+                        height: '1rem',
+                        width:
+                          index % 5 === 0
+                            ? '66%'
+                            : index % 5 === 1 || index % 5 === 4
+                              ? '75%'
+                              : index % 5 === 2
+                                ? '60%'
+                                : '83%',
+                        borderRadius: '0.25rem',
+                        animationName: 'tw-keyframes-glow',
```

Comment:
I am advocating for removing this loading logic, but I feel it's important to mention that this code lives in `space-kit` which has no knowledge of our tailwind classes, so `tw-keyframes-glow` will have no effect unless this is used in engine-frontend.

Reply:
Since we already have all the animation logic in engine, lets keep it there!